packages/solicitudes/src/use-cases/institucion/update.institucion.use-cases.js

```diff
@@ -0,0 +1,11 @@
+const { Logger } = require('@siiges-services/shared');
+
+const update = (updateQuery) => async (identifierObj, changes) => {
+  Logger.info('[solicitudes/update]: Updating solicitud');
+  const solicitudUpdated = await updateQuery(identifierObj, changes);
+  Logger.info('[solicitudes/update] A solicitud was updated');
+
+  return solicitudUpdated;
+};
+
+module.exports = update;
```

Comment:
Why are you using this use-cases about update institicion and solicitud at the same time?
This are related to diligences?